inference-engine/src/cldnn_engine/cldnn_executable_network.cpp

```diff
@@ -134,6 +139,7 @@ InferenceEngine::Parameter CLDNNExecNetwork::GetMetric(const std::string &name)
 }
 
 RemoteContext::Ptr CLDNNExecNetwork::GetContext() const {
+    OV_ITT_SCOPED_TASK(itt::domains::CLDNNPlugin, "CLDNNExecNetwork::GetContext");
```

Comment:
Not sure for which specific task you are instrumneting the clDNN, but wrapping EVERY function (especially setters/getters that may be called many many times) seems redundant. At least for the perf debugging. Myabe for conditional compilation this is needed...

Reply:
fixed